TradeItIosTicketSDK2/ActivityIndicatorFactory.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+@objc public protocol ActivityIndicatorFactory {
+    func build(frame: CGRect) -> UIView
+}
+
+class DefaultActivityIndicatorFactory: ActivityIndicatorFactory {
+    func build(frame: CGRect) -> UIView {
+        let view = UIActivityIndicatorView(frame: frame)
+        view.color = UIColor.gray
+        view.startAnimating()
+        return view
```

Comment:
`UIActivityIndicatorView` doesn't adjust the spinner size based on what frame you pass in so it just shows a small spinner anyway.